themes/engineering/.circleci/config.yml

```diff
@@ -0,0 +1,43 @@
+version: 2

```

Comment:
I think we can delete this; we have a circleci configuration in the root of the repo, and since this isn't a separate repo, it's never going to be read.